### PR TITLE
new combined b-discriminator 

### PR DESCRIPTION
**Pull request by jpata**
Implements the new combined b-discriminator (https://github.com/cms-sw/cmssw/pull/10197) offline.
- I needed to re-calculate the soft muon and electron taggers: this is done by changing the TagInfo inputs and corresponding statements in `combined_cmssw.py`
- the jet probability taggers are switched to `pf*`-based evaluation
- the old `AdditionalBTag.py` module has been repurposed to evaluate the new `cMVAv2`

The new b-tagger is saved in `Jet_btagBDT`.

This also solves issue https://github.com/vhbb/cmssw/issues/141.

### CODE REVIEW

File: VHbbAnalysis/Heppy/test/vhbb_combined.py [hunk 1]
```diff
@@ -82,11 +82,11 @@
...
 # 
-# btagana=cfg.Analyzer(
-#     verbose=False,
-#     class_object=AdditionalBTag,
-# )
-# sequence.insert(sequence.index(VHbb),btagana)
+btagana=cfg.Analyzer(
+    verbose=False,
+    class_object=AdditionalBTag,
+)
```
Comment (arizzi) [inline VHbbAnalysis/Heppy/test/vhbb_combined.py]: we should uncomment line 9 I guess? @jpata

**Comment by jpata** [comment]
oops, not sure how I missed it because it worked in testing. let me try again (new PR tomorrow)

**Comment by arizzi** [comment]
no PR needed, I'll fix on my side

**Comment by jpata** [comment]
ok thanks. My validation scripts also test the new BDT.
